announcements/move-to-cluster-api.md

```diff
@@ -0,0 +1,42 @@
+
+### Why is Giant Swarm moving to Cluster API?
+
+In 2017, due to a lack of better alternatives, Giant Swarm decided to build their product based on their own implementation of automation, tools, and operators. This allowed Giant Swarm to provide a stable and easy-to-use product that you are currently using to manage your Kubernetes clusters. Although this solution was ideal for a long time it has its downsides:
+- You have limited control over configuration of individual parts of the infrastructure. The product is not flexible enough and sometimes doesn't support all of your use cases.
+- Adding new functionality is time-consuming and as hyperscalers (Amazon, Microsoft) expand their offering it becomes increasingly difficult to keep up and make it all available to you in a timely manner.
+- Your cluster management is limited to what Giant Swarm tooling enables and there are no available alternatives.
+- Although Giant Swarm provides pure "vanilla" Kubernetes clusters, it is currently not possible to move the workload clusters and have them managed by another product without migration. 
+
+Giant Swarm has been closely watching the development of the upstream Cluster API project and since it achieved a satisfying level of maturity, decided to adopt it in order to address all of the downsides mentioned above and bring more value to you:
+- Significantly more flexibility and configurability over your infrastructure. Things like deploying into existing networks or ... will now be possible.
+- A future-proof API that protects you from vendor lock-ins, developed by a dynamically growing community which will increase the speed of adding new functionality over time.
+- The possibility to use all tooling compatible with the Cluster API -- not only that created by Giant Swarm.
+- Ability to create and manage workload clusters with managed control planes like EKS or AKS.
+- Adding new providers will become easier and Giant Swarm will be in a better position to expand their offering beyond AWS, Azure, and current On-Premises solutions.
+- The possibility to have workload clusters from different providers managed by one management cluster to enable multi-cloud use cases.
+
+### What will the move look like?
+
+Giant Swarm is investing an enormous amount of effort to make the process safe and convenient for you. From your perspective, it will look like a regular update to a new major version -- no migration will be required. On the Giant Swarm side, a series of well-tested, automated actions will be performed to transform your current clusters into the Cluster API ones. Giant Swarm wants to provide a transition path for the clusters in all versions and transform them into Cluster API clusters with little to no downtime in the Kubernetes control plane and no downtime in your workloads. Giant Swarm will perform an in-place upgrade to preserve your current IP addresses.
```

Comment:
> no migration will be required

I think this wording is confusing. Within GS we call a move between two clusters a "migration" but similarly moving from our controllers to upstream controllers could be called a "migration" (as the team name we chose for the taskforce on this also suggests). Maybe we can be more clear here, like:

`no migration will be needed, i.e. this will be an in-place upgrade of your clusters.`

Reply:
I know you mention it later and explain it in detail, but smh I feel the word migration still sticks out while reading, maybe we could also just remove or replace it

Reply:
I was trying hard to avoid calling it a "migration" throughout the document, I'm calling it "transition "move" etc. just for this reason to not scare customers that they have to migrate. Does it make sense? I could change it to:

> no workload migration will be needed, this will be an in-place upgrade of your clusters.

Reply:
```suggestion
Giant Swarm is investing an enormous amount of effort to make the process safe and convenient for you. From your perspective, it will look like a regular update to a new major version -- no workload migration will be required. On the Giant Swarm side, a series of well-tested, automated actions will be performed to transform your current clusters into the Cluster API ones. Giant Swarm wants to provide a transition path for the clusters in all versions and transform them into Cluster API clusters with little to no downtime in the Kubernetes control plane and no downtime in your workloads. Giant Swarm will perform an in-place upgrade to preserve your current IP addresses.
```

Actually, you just adding workload there makes it clear enough, we mention in-place later on anyway